src/components/ComposerAttachments.vue

```diff
@@ -22,17 +22,22 @@
 
 <template>
 	<div class="new-message-attachments">
-		<ul>
-			<li v-for="attachment in value" :key="attachment.id">
-				<div class="new-message-attachment-name">
-					{{ attachment.displayName ? attachment.displayName : attachment.fileName }}
-				</div>
-				<div class="new-message-attachments-action svg icon-delete" @click="onDelete(attachment)" />
-			</li>
-			<li v-if="uploading" class="attachments-upload-progress">
-				<div :class="{'icon-loading-small': uploading}" />
-				<div>{{ uploading ? t('mail', 'Uploading {percent}% …', {percent: uploadProgress}) : '' }}</div>
-			</li>
+		<div v-if="hasNextLine" class="message-attachments-count" @click="isToggle = !isToggle">
+			<span>
+				{{ n('mail', '{count} attachment', '{count} attachments', attachments.length, { count: attachments.length }) }} ({{ formatBytes(totalSizeOfUpload()) }})
+			</span>
+			<ChevronDown v-if="isToggle" :size="24" />
+			<ChevronUp v-if="!isToggle" :size="24" />
```

Comment:
```suggestion
			<ChevronUp v-if="isToggle" :size="24" />
			<ChevronDown v-if="!isToggle" :size="24" />
```
To make it consistent with the mockups.